imageroot/events/certificate-updated/00validate_event

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
```

Comment:
Why not merging 00validate_event with 20install_certificate? It should be more readable and easier to maintain.

Reply:
A matter of taste... IMO Python is good to parse JSON, Bash is more readable when it's just a matter of invoking other commands. Also splitting into two steps writes a bit more of log information for free.

Reply:
A matter of taste... IMO Python is good to parse JSON, Bash is more readable when it's just a matter of invoking other commands. Also splitting into two steps writes a bit more of log information for free.

Reply:
And adds 20-30 of code lines :-1: